configs/a2c_gptj.yml

```diff
@@ -0,0 +1,55 @@
+train:
```

Comment:
Remove the `a2c_gptj.yml` unless tested thoroughly. We don't want folks wasting their compute resources on a large-ish tune with untested hparams.